recipes/foxi/all/conanfile.py

```diff
@@ -0,0 +1,74 @@
+from conans import ConanFile, CMake, tools
+import glob
+import os
+import shutil
+
+
+class FoxiConan(ConanFile):
+    name = "foxi"
+    description = "ONNXIFI with Facebook Extension."
+    license = "MIT"
+    topics = ("conan", "foxi", "onnxifi")
+    homepage = "https://github.com/houseroad/foxi"
+    url = "https://github.com/conan-io/conan-center-index"
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"fPIC": [True, False]}
+    default_options = {"fPIC": True}
+
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake"
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = glob.glob("foxi-*")[0]
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def _patch_sources(self):
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            tools.patch(**patch)
+        cmakelists = os.path.join(self._source_subfolder, "CMakeLists.txt")
+        tools.replace_in_file(cmakelists, "add_msvc_runtime_flag(foxi_loader)", "")
+        tools.replace_in_file(cmakelists, "add_msvc_runtime_flag(foxi_dummy)", "")
+        tools.replace_in_file(cmakelists,
+                              "DESTINATION lib",
+                              "RUNTIME DESTINATION bin ARCHIVE DESTINATION lib LIBRARY DESTINATION lib")
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["FOXI_WERROR"] = False
+        self._cmake.configure()
```

Comment:
Build folder?

Reply:
There is no strong reason to use out-of-source build, in-source build just works.